service/static/index.html

```diff
@@ -0,0 +1,108 @@
+                </div>
+                <div class="col-sm-4">
+                  <button type="submit" class="btn btn-primary" id="retrieve-btn">Retrieve</button>
+                  <button type="submit" class="btn btn-danger" id="delete-btn">Delete</button>
+                  &nbsp;&nbsp;&nbsp;
+                  <a href="items.html">
+                    <button>Item Page</button>
+                  </a>
+                </div>
+              </div>
+            </div> <!-- form div -->
+
+            <div class="form-horizontal">
+              <div class="form-group">
+                <label class="control-label col-sm-2" for="wishlist_name">Name:</label>
+                <div class="col-sm-10">
+                  <input type="text" class="form-control" id="wishlist_name" placeholder="Enter name for Wishlist">
+                </div>
+              </div>
+              <div class="form-group">
+                <label class="control-label col-sm-2" for="wishlist_user_id">User ID:</label>
+                <div class="col-sm-10">
+                  <input type="text" class="form-control" id="wishlist_user_id" placeholder="Enter user id for Wishlist">
+                </div>
+              </div>
+              <div class="form-group">
+                <label class="control-label col-sm-2" for="wishlist_items">Items:</label>
+                <div class="col-sm-10">
+                  <input type="text" class="form-control" id="wishlist_items" placeholder="Enter items for Wishlist in format [item1, item2, ...]">
+                </div>
+              </div>
+              <div class="form-group">
+                <label class="control-label col-sm-2" for="wishlist_status">Status:</label>
+                <div class="col-sm-10">
+                  <select class="form-control" id="wishlist_status">
+                      <option value="true" selected>True</option>
+                      <option value="false">False</option>
+                  </select>
+                </div>
+              </div>
+              <div class="form-group">
+                <div class="col-sm-offset-2 col-sm-10">
+                  <button type="submit" class="btn btn-primary" id="search-btn">Search</button>
+                  <button type="submit" class="btn btn-primary" id="clear-btn">Clear</button>
+                  <button type="submit" class="btn btn-success" id="create-btn">Create</button>
+                  <button type="submit" class="btn btn-warning" id="update-btn">Update</button>
+                </div>
+              </div>
+          </div> <!-- form div -->
+        </div>
+
+        <!-- Search Results -->
+        <div class="table-responsive col-md-10" id="search_results">
+          <table class="table-striped">
+            <thead>
+            <tr>
+                <th class="col-md-2">ID</th>
+                <th class="col-md-2">Name</th>
+                <th class="col-md-2">UserID</th>
+                <th class="col-md-2">Items</th>
```

Comment:
I don't think we should have items. How will we enter items in the text field? Maybe wishlist UI can create a wishlist with no items, and then simply use the items UI to run add items?

Reply:
How do you wanna display items in the wishlist page?

Reply:
You don't. We can see the wishlists on the wishlist page.... We can go to the items page to see the items I guess.

I made the status fix, works fine.